windows/d2-installer.nsi

```diff
@@ -351,6 +352,15 @@ SectionGroupEnd
 Function DetectVSAndSDK
     ClearErrors
 
+    ReadRegStr $0 HKLM "SOFTWARE\Microsoft\VisualStudio\SxS\VS7" "15.0"
```

Comment:
I have installed VS 2017 Pro and there is no key HKLM\SOFTWARE\Microsoft\VisualStudio\SxS\VS7.
But there is HKLM\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\SxS\VS7.